qemu/tests/cfg/block_hotplug.cfg

```diff
@@ -136,6 +142,9 @@
                 controller_model = virtio-scsi-pci
                 s390x:
                     controller_model = virtio-scsi-ccw
+                q35:
+                    pci_bus_stg0 = rootport1
+                    pci_bus_stg1 = rootport2
```

Comment:
could you explain why only set pci_bug_stg* to virtio_blk?

Reply:
@CongLi Yes, this variants is for a virtio-scsi block plugged into a vm boot up with only virtio-blk-pci(virtio_blk:). Originally, it plug a virtio-scsi-pci first, then the scsi-hd. For q35, need to move virtio-scsi-pci to root-port, so here I define pci_bus for stg0 and stg1. Since scsi-hd is plugged to the virtio-scsi-pci instead of pci.0/pcie.0, so I shift `pci_bus=None` after it is set for the controller in line 122 on block_hotplug.py, which means the bus parameter will not be set to the device scsi-hd again. Thanks for review.

Reply:
thanks for the explanation.